buildkite/scripts/version-linter-patch-missing-type-shapes.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -eox pipefail
+
+if [[ $# -ne 1 ]]; then
+    echo "Usage: $0 <release-branch>"
+    exit 1
+fi
+
+git config --global --add safe.directory /workdir
+
+source buildkite/scripts/handle-fork.sh
+source buildkite/scripts/export-git-env-vars.sh
+
+release_branch=${REMOTE}/$1
```

Comment:
This seems wrong. Why is `REMOTE` here?

Reply:
REMOTE env var is set in `handle-fork.sh`. This was implemented in order to allow run CI on forks (needed in umt/itn).

If we wish to disable running Ci on forks, i'm more than happy to remove both env var and handle-fork.sh call here